### PR TITLE
Creating RU version of getting_started_introduction

### PR DESCRIPTION
**Pull request by greggubarev**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by noroadsleft** [comment]
Requesting reviews from @StrikerRUS and/or @nabokovas.

File: docs/ru-ru/getting_started_introduction.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Введение
+
+Эта страница пытается объяснить основную информацию, которую вы должны знать, чтобы работать с проектом QMK. Предполагается, что вы знакомы с навигацией в оболочке Unix, но не предполагаете, что вы знакомы с C или с компиляцией с использованием make.
+
+## Базовая структура QMK
+
+QMK это форк [Jun Wako](https://github.com/tmk) [tmk_keyboard](https://github.com/tmk/tmk_keyboard) проекта. Оригинальный код TMK с изменениями можно найти в папке `tmk`. Дополнения QMK к проекту можно найти в папке «Quant». Проекты клавиатуры можно найти в папках `handwired` и `keyboard`.
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
QMK - это форк [Джуна Вако (Jun Wako)](https://github.com/tmk) проекта [tmk_keyboard](https://github.com/tmk/tmk_keyboard). Оригинальный код TMK с изменениями можно найти в папке `tmk`. Дополнения QMK к проекту можно найти в папке `quantum`. Проекты клавиатур можно найти в папках `handwired` и `keyboard`.
```

File: docs/ru-ru/getting_started_introduction.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+### Структура пространства пользователя
+
+Внутри папки `users` находится каталог для каждого пользователя. Это место для пользователей, чтобы поместить код, который они могли бы использовать между клавиатурами. Для получения дополнительной информации см. документацию для [функций пользовательского пространства](feature_userspace.md).
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
Внутри папки `users` находится каталог для каждого пользователя. Это место для пользователей, куда они могут поместить код, чтобы использовать его с разными клавиатурами. Для получения дополнительной информации обратитесь к документации по [функциям пользовательского пространства](feature_userspace.md).
```

File: docs/ru-ru/getting_started_introduction.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+### Структура проекта клавиатуры
+
+Внутри папки `keyboards`, есть подпапки `handwired` и есть подкаталоги поставщиков и производителей, для примера, `clueboard` - это каталог для каждого проекта клавиатуры, например, `qmk_firmware/keyboards/clueboard/2x1800`. В нем вы найдете следующую структуру:
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
Внутри папки `keyboards` есть подпапки `handwired` и есть подкаталоги поставщиков и производителей, для примера, `clueboard` - это каталог для каждого проекта клавиатуры, например, `qmk_firmware/keyboards/clueboard/2x1800`. В нем вы найдете следующую структуру:
```

File: docs/ru-ru/getting_started_introduction.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+* `keymaps/`: Различные раскладки клавиш, которые можно построить
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* `keymaps/`: Различные раскладки клавиш, которые можно собрать
```

File: docs/ru-ru/getting_started_introduction.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+* `rules.mk`: Файл, который устанавливает параметры по умолчанию "make". Не редактируйте этот файл напрямую, вместо этого используйте специальную раскладку `rules.mk`.
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* `rules.mk`: Файл, который устанавливает параметры по умолчанию для команды "make". Не редактируйте этот файл напрямую, вместо этого используйте `rules.mk`, относящийся к конкретной раскладке.
```

File: docs/ru-ru/getting_started_introduction.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+* `config.h`: Файл, который устанавливает параметры времени компиляции по умолчанию. Не редактируйте этот файл напрямую, вместо этого используйте специальную раскладку `config.h`.
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* `config.h`: Файл, который устанавливает параметры времени компиляции по умолчанию. Не редактируйте этот файл напрямую, вместо этого используйте `config.h`, относящийся к конкретной раскладке..
```

File: docs/ru-ru/getting_started_introduction.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+* `info.json`: Файл, используемый для настройки макета для QMK Configurator. Смотрите [Поддержку конфигуратора](reference_configurator_support.md) для дополнительной информации.
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* `info.json`: Файл настройки раскладки для QMK Configurator. Посмотрите [Поддержку конфигуратора](reference_configurator_support.md) для дополнительной информации.
```

File: docs/ru-ru/getting_started_introduction.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+* `readme.md`: Краткий обзор клавиатуры.
+* `<keyboardName>.h`: его файл находится там, где раскладка клавиатуры определяется по матрице переключателей клавиатуры.
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* `<keyboardName>.h`: В этом файле определяется раскладка клавиатуры по матрице переключателей клавиатуры.
```

File: docs/ru-ru/getting_started_introduction.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+* `<keyboardName>.c`: Этот файл, где вы можете найти пользовательский код для клавиатуры.
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* `<keyboardName>.c`: В этом файле вы можете найти пользовательский код для клавиатуры.  
```

File: docs/ru-ru/getting_started_introduction.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+* `keymaps/`: Различные раскладки клавиш, которые можно построить
+* `rules.mk`: Файл, который устанавливает параметры по умолчанию "make". Не редактируйте этот файл напрямую, вместо этого используйте специальную раскладку `rules.mk`.
+* `config.h`: Файл, который устанавливает параметры времени компиляции по умолчанию. Не редактируйте этот файл напрямую, вместо этого используйте специальную раскладку `config.h`.
+* `info.json`: Файл, используемый для настройки макета для QMK Configurator. Смотрите [Поддержку конфигуратора](reference_configurator_support.md) для дополнительной информации.
+* `readme.md`: Краткий обзор клавиатуры.
+* `<keyboardName>.h`: его файл находится там, где раскладка клавиатуры определяется по матрице переключателей клавиатуры.
+* `<keyboardName>.c`: Этот файл, где вы можете найти пользовательский код для клавиатуры.  
+
+Для получения дополнительной информации о структуре проекта см. [QMK Руководство по клавиатуре](hardware_keyboard_guidelines.md).
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
Для получения дополнительной информации о структуре проекта обратитесь к [Руководству QMK по клавиатуре](hardware_keyboard_guidelines.md).
```

File: docs/ru-ru/getting_started_introduction.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+
+### Структура раскладки клавиатуры
+
+В каждой папке раскладки клавиатуры могут быть найдены следующие файлы. Только `keymap.c` требуется, и если остальные файлы не найдены, будут выбраны параметры по умолчанию.
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
В каждой папке раскладки клавиатуры могут быть найдены следующие файлы. Обязательным является только файл `keymap.c`, и если остальные файлы не найдены, то будут выбраны параметры по умолчанию.
```

File: docs/ru-ru/getting_started_introduction.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+
+* `config.h`: варианты настройки вашей раскладки клавиатуры
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* `config.h`: настройки вашей раскладки клавиатуры
```

File: docs/ru-ru/getting_started_introduction.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+* `keymap.c`: необходим весь код вашей раскладки клавиатуры
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* `keymap.c`: весь код вашей раскладки клавиатуры (обязателен)
```

File: docs/ru-ru/getting_started_introduction.md [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+* `rules.mk`: функции QMK, которые включены
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* `rules.mk`: активированные функции QMK
```

File: docs/ru-ru/getting_started_introduction.md [hunk 15]
```diff
@@ -0,0 +1,58 @@
...
+* `readme.md`: описание вашей раскладки клавиш, как ее могут использовать другие, и объяснения функций. Пожалуйста, загрузите изображения на сервис, как imgur.
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* `readme.md`: описание вашей раскладки клавиш, как ее могут использовать другие, и объяснения функций. Пожалуйста, загрузите изображения на сервис, такой как imgur.
```

File: docs/ru-ru/getting_started_introduction.md [hunk 16]
```diff
@@ -0,0 +1,58 @@
...
+
+# The `config.h` файл
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
# Файл `config.h`
```

File: docs/ru-ru/getting_started_introduction.md [hunk 17]
```diff
@@ -0,0 +1,58 @@
...
+
+Существует 3 возможных местоположения `config.h`:
+
+* keyboard (`/keyboards/<keyboard>/config.h`)
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* клавиатура (`/keyboards/<keyboard>/config.h`)
```

File: docs/ru-ru/getting_started_introduction.md [hunk 18]
```diff
@@ -0,0 +1,58 @@
...
+* userspace (`/users/<user>/config.h`)
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* пространство пользователя (`/users/<user>/config.h`)
```

File: docs/ru-ru/getting_started_introduction.md [hunk 19]
```diff
@@ -0,0 +1,58 @@
...
+* keymap (`/keyboards/<keyboard>/keymaps/<keymap>/config.h`)
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* раскладка клавиш (`/keyboards/<keyboard>/keymaps/<keymap>/config.h`)
```

File: docs/ru-ru/getting_started_introduction.md [hunk 20]
```diff
@@ -0,0 +1,58 @@
...
+
+Система сборки автоматически подбирает файлы конфигурации в указанном выше порядке. Если вы хотите переопределить любую настройку, заданную предыдущим `config.h`, вам сначала нужно будет включить некоторый шаблонный код для настроек, которые вы хотите изменить.
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
Система сборки автоматически загружает файлы конфигурации в указанном выше порядке. Если вы хотите переопределить любую настройку, заданную предыдущим `config.h`, вам сначала нужно будет включить некоторый шаблонный код для настроек, которые вы хотите изменить.
```

File: docs/ru-ru/getting_started_introduction.md [hunk 21]
```diff
@@ -0,0 +1,58 @@
...
+
+```
+#pragma once
+```
+
+Затем, чтобы переопределить настройку из предыдущего файла `config.h`, вы должны `#undef` и затем `#define` снова.
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
Затем, чтобы переопределить настройку из предыдущего файла `config.h`, вы должны сделать `#undef` и `#define` для неё снова.
```

File: docs/ru-ru/getting_started_introduction.md [hunk 22]
```diff
@@ -0,0 +1,58 @@
...
+
+Код и настройка шаблона вместе выглядят так:
+```
+#pragma once
+
+// Переопределения здесь!
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
// Переопределения производятся здесь!
```

**Comment by greggubarev** [comment]
> Спасибо за перевод! Оставил несколько исправлений ниже.

@StrikerRUS thanks for the help!

**Comment by StrikerRUS** [comment]
@greggubarev Thanks for committing change requests! However, there are 13 more ones remained. Click to ``Load more...`` to see them.

image

**Comment by greggubarev** [comment]
> @greggubarev Thanks for committing change requests! However, there are 13 more ones remained. Click to `Load more...` to see them.
> 
> image

@StrikerRUS got it, sorry)))

**Comment by noroadsleft** [comment]
@StrikerRUS Can you leave a PR approval if you're good with this, please?

File: docs/ru-ru/getting_started_introduction.md [hunk 23]
```diff
@@ -0,0 +1,58 @@
...
+QMK - это форк [Джуна Вако (Jun Wako)](https://github.com/tmk) проекта [tmk_keyboard](https://github.com/tmk/tmk_keyboard). Оригинальный код TMK с изменениями можно найти в папке `tmk`. Дополнения QMK к проекту можно найти в папке `quantum`. Проекты клавиатур можно найти в папках `handwired` и `keyboard`.
+
+### Структура пространства пользователя
+
+Внутри папки `users` находится каталог для каждого пользователя. Это место для пользователей, куда они могут поместить код, чтобы использовать его с разными клавиатурами. Для получения дополнительной информации обратитесь к документации по [функциям пользовательского пространства](feature_userspace.md).
+
+### Структура проекта клавиатуры
+
+Внутри папки `keyboards` есть подпапки `handwired` и есть подкаталоги поставщиков и производителей, для примера, `clueboard` - это каталог для каждого проекта клавиатуры, например, `qmk_firmware/keyboards/clueboard/2x1800`. В нем вы найдете следующую структуру:
+* `keymaps/`: Различные раскладки клавиш, которые можно собрать
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* `keymaps/`: Различные раскладки клавиш, которые можно собрать.
```

File: docs/ru-ru/getting_started_introduction.md [hunk 24]
```diff
@@ -0,0 +1,58 @@
...
+
+Эта страница пытается объяснить основную информацию, которую вы должны знать, чтобы работать с проектом QMK. Предполагается, что вы знакомы с навигацией в оболочке Unix, но не предполагается, что вы знакомы с C или с компиляцией с использованием make.
+
+## Базовая структура QMK
+
+QMK - это форк [Джуна Вако (Jun Wako)](https://github.com/tmk) проекта [tmk_keyboard](https://github.com/tmk/tmk_keyboard). Оригинальный код TMK с изменениями можно найти в папке `tmk`. Дополнения QMK к проекту можно найти в папке `quantum`. Проекты клавиатур можно найти в папках `handwired` и `keyboard`.
+
+### Структура пространства пользователя
+
+Внутри папки `users` находится каталог для каждого пользователя. Это место для пользователей, куда они могут поместить код, чтобы использовать его с разными клавиатурами. Для получения дополнительной информации обратитесь к документации по [функциям пользовательского пространства](feature_userspace.md).
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
Внутри папки `users` находится каталог для каждого пользователя. Это место для пользователей, куда они могут поместить код, чтобы использовать его с разными клавиатурами. Для получения дополнительной информации обратитесь к документации по [функциям пользовательского пространства](ru-ru/feature_userspace.md).
```

File: docs/ru-ru/getting_started_introduction.md [hunk 25]
```diff
@@ -0,0 +1,58 @@
...
+
+### Структура проекта клавиатуры
+
+Внутри папки `keyboards` есть подпапки `handwired` и есть подкаталоги поставщиков и производителей, для примера, `clueboard` - это каталог для каждого проекта клавиатуры, например, `qmk_firmware/keyboards/clueboard/2x1800`. В нем вы найдете следующую структуру:
+* `keymaps/`: Различные раскладки клавиш, которые можно собрать
+* `rules.mk`: Файл, который устанавливает параметры по умолчанию для команды "make". Не редактируйте этот файл напрямую, вместо этого используйте `rules.mk`, относящийся к конкретной раскладке.
+* `config.h`: Файл, который устанавливает параметры времени компиляции по умолчанию. Не редактируйте этот файл напрямую, вместо этого используйте `config.h`, относящийся к конкретной раскладке..
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* `config.h`: Файл, который устанавливает параметры времени компиляции по умолчанию. Не редактируйте этот файл напрямую, вместо этого используйте `config.h`, относящийся к конкретной раскладке.
```

File: docs/ru-ru/getting_started_introduction.md [hunk 26]
```diff
@@ -0,0 +1,58 @@
...
+* `info.json`: Файл настройки раскладки для QMK Configurator. Посмотрите [Поддержку конфигуратора](reference_configurator_support.md) для дополнительной информации.
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* `info.json`: Файл настройки раскладки для QMK Configurator. Посмотрите [Поддержку конфигуратора](ru-ru/reference_configurator_support.md) для дополнительной информации.
```

File: docs/ru-ru/getting_started_introduction.md [hunk 27]
```diff
@@ -0,0 +1,58 @@
...
+* `readme.md`: Краткий обзор клавиатуры.
+* `<keyboardName>.h`: В этом файле определяется раскладка клавиатуры по матрице переключателей клавиатуры.
+* `<keyboardName>.c`: В этом файле вы можете найти пользовательский код для клавиатуры.  
+
+Для получения дополнительной информации о структуре проекта обратитесь к [Руководству QMK по клавиатуре](hardware_keyboard_guidelines.md).
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
Для получения дополнительной информации о структуре проекта обратитесь к [Руководству QMK по клавиатуре](ru-ru/hardware_keyboard_guidelines.md).
```

File: docs/ru-ru/getting_started_introduction.md [hunk 28]
```diff
@@ -0,0 +1,58 @@
...
+### Структура проекта клавиатуры
+
+Внутри папки `keyboards` есть подпапки `handwired` и есть подкаталоги поставщиков и производителей, для примера, `clueboard` - это каталог для каждого проекта клавиатуры, например, `qmk_firmware/keyboards/clueboard/2x1800`. В нем вы найдете следующую структуру:
+* `keymaps/`: Различные раскладки клавиш, которые можно собрать
+* `rules.mk`: Файл, который устанавливает параметры по умолчанию для команды "make". Не редактируйте этот файл напрямую, вместо этого используйте `rules.mk`, относящийся к конкретной раскладке.
+* `config.h`: Файл, который устанавливает параметры времени компиляции по умолчанию. Не редактируйте этот файл напрямую, вместо этого используйте `config.h`, относящийся к конкретной раскладке..
+* `info.json`: Файл настройки раскладки для QMK Configurator. Посмотрите [Поддержку конфигуратора](reference_configurator_support.md) для дополнительной информации.
+* `readme.md`: Краткий обзор клавиатуры.
+* `<keyboardName>.h`: В этом файле определяется раскладка клавиатуры по матрице переключателей клавиатуры.
+* `<keyboardName>.c`: В этом файле вы можете найти пользовательский код для клавиатуры.
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* `<keyboardName>.c`: В этом файле вы можете найти пользовательский код для клавиатуры.
```

File: docs/ru-ru/getting_started_introduction.md [hunk 29]
```diff
@@ -0,0 +1,58 @@
...
+* `<keyboardName>.h`: В этом файле определяется раскладка клавиатуры по матрице переключателей клавиатуры.
+* `<keyboardName>.c`: В этом файле вы можете найти пользовательский код для клавиатуры.  
+
+Для получения дополнительной информации о структуре проекта обратитесь к [Руководству QMK по клавиатуре](hardware_keyboard_guidelines.md).
+
+### Структура раскладки клавиатуры
+
+В каждой папке раскладки клавиатуры могут быть найдены следующие файлы. Обязательным является только файл `keymap.c`, и если остальные файлы не найдены, то будут выбраны параметры по умолчанию.
+
+* `config.h`: настройки вашей раскладки клавиатуры
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* `config.h`: настройки вашей раскладки клавиатуры.
```

File: docs/ru-ru/getting_started_introduction.md [hunk 30]
```diff
@@ -0,0 +1,58 @@
...
+* `keymap.c`: весь код вашей раскладки клавиатуры (обязателен)
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* `keymap.c`: весь код вашей раскладки клавиатуры (обязателен).
```

File: docs/ru-ru/getting_started_introduction.md [hunk 31]
```diff
@@ -0,0 +1,58 @@
...
+* `rules.mk`: активированные функции QMK
```
Comment (StrikerRUS) [inline docs/ru-ru/getting_started_introduction.md]:
```suggestion
* `rules.mk`: активированные функции QMK.
```

**Comment by greggubarev** [comment]
> @greggubarev Please fix internal links and some punctuation.

thanks for fix

**Comment by greggubarev** [comment]
> @greggubarev Please fix internal links and some punctuation.

**Comment by greggubarev** [comment]
@nabokovas can you please review this PR?

**Comment by greggubarev** [comment]
Guys, can i get feedback about this PR? @noroadsleft @StrikerRUS @nabokovas

**Comment by noroadsleft** [comment]
Sorry for losing track of this, @greggubarev. Спасибо!

**Comment by greggubarev** [comment]
@noroadsleft can you change please the label - from "Awaiting review" to "hacktoberfest" (as https://github.com/qmk/qmk_firmware/pull/7226)? For qualify a QMK t-shirt

**Comment by noroadsleft** [comment]
> @noroadsleft can you change please the label - from "Awaiting review" to "hacktoberfest" (as #7226)? For qualify a QMK t-shirt

Done; forgot to check the labels here.

[Claim form for the shirt is here](https://docs.google.com/forms/d/e/1FAIpQLSeFt0mmW2e8tFgTsGGlSUET_8dpGiyuOTZKyS5g2jcIM20d-Q/viewform), though we decided one translation PR should be enough to qualify. Fill this out ASAP; I'm not sure where Jack is on the shirt order.

**Comment by greggubarev** [comment]
> > @noroadsleft can you change please the label - from "Awaiting review" to "hacktoberfest" (as #7226)? For qualify a QMK t-shirt
> 
> Done; forgot to check the labels here.
> 
> [Claim form for the shirt is here](https://docs.google.com/forms/d/e/1FAIpQLSeFt0mmW2e8tFgTsGGlSUET_8dpGiyuOTZKyS5g2jcIM20d-Q/viewform), though we decided one translation PR should be enough to qualify. Fill this out ASAP; I'm not sure where Jack is on the shirt order.

Thank you! it will be a pity if I do not get a shirt ((

**Pull request merged**